src/stories/design-decision.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Divider, Typography } from '@material-ui/core';
+
+import DoorButton from './BaseComponents/door-buttons';
+import LiftButton from './BaseComponents/lift-buttons';
+import RobotButton from './BaseComponents/robot-buttons';
+import DispenserButton from './BaseComponents/dispenser-buttons';
+import {
+  StyleTyping,
+  doors,
+  doorStates,
+  lifts,
+  liftStates,
+  robotStates,
+  dispenserStates,
+  defaultStyles,
+} from './BaseComponents/Utils';
+
+export default {
+  title: 'Design Decisions',
+};
+
+const styles: StyleTyping = {
+  ...defaultStyles,
+  example: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '1rem 0',
+  },
+  ellipsis: {
+    border: '1px solid black',
+    padding: '0.5rem',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    maxWidth: '10rem',
+  },
+  explanation: {
+    margin: '1rem 0',
+  },
+  panelButton: {
+    border: '2px solid #CCCCCC',
+    padding: 5,
+    width: '4rem',
+    textAlign: 'center',
+    borderRadius: '4px',
+  },
+};
+
+export const HandleLongName = () => (
```

Comment:
This still uses hardcoded values, either refactor to have a single source of truth for the design of long labels or render a door/lift item with long names.